paths/custom_metadata_properties/update.yaml

```
@@ -8,40 +8,40 @@ parameters:
 - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
 - "$ref": "../../parameters.yaml#/account_id"
 - "$ref": "../../parameters.yaml#/id"
-- description: name of the property
-  example:
-  - Fruit
-  name: name
-  in: query
-  schema:
-    type: string
-- description: description of property
-  example:
-  - A healthy snack for all ages
-  name: description
-  in: query
-  schema:
-    type: string
-- description: ids of projects that the property belongs to
-  example:
-    - abcd1234cdef1234abcd1234cdef1234
-  name: project_ids
-  in: query
-  schema:
-    type: array
-    items:
-      type: string
-- description: value options of property (only applies to single or multi select properties)
-  example:
-    - Apple
-    - Banana
-    - Coconut
-  name: value_options
-  in: query
-  schema:
-    type: array
-    items:
-      type: string
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
```

Should we also add the required `name` and `datatype` section?
```suggestion
      schema:
        required:
        - name
        - data_type
```

---

yeah, I haven't checked the backend code, but I would guess that the data type is not changeable once the property is created, and the name is certainly not required, if one doesn't want to change it.